qdrant-landing/content/documentation/concepts/storage.md

```diff
@@ -7,37 +7,43 @@ aliases:
 
 # Storage
 
-All data within one collection is divided into segments.
-Each segment has its independent vector and payload storage as well as indexes.
+A Qdrant collection can be split into multiple shards. Each shard is a separate storage unit. By default, a collection is created with a single [shard](/documentation/guides/distributed_deployment/). They are used to distribute data across nodes in a cluster, enabling parallel processing and improving performance. 
 
-Data stored in segments usually do not overlap.
-However, storing the same point in different segments will not cause problems since the search contains a deduplication mechanism.
+From a storage perspective, data within a shard is organized into segments. Each segment independently manages its vector and payload storage, along with their respective indexes.
 
-The segments consist of vector and payload storages, vector and payload [indexes](/documentation/concepts/indexing/), and id mapper, which stores the relationship between internal and external ids.
+![Storage Architecture](/documentation/concepts/storage/storage-architecture.png)
 
-A segment can be `appendable` or `non-appendable` depending on the type of storage and index used.
-You can freely add, delete and query data in the `appendable` segment.
-With `non-appendable` segment can only read and delete data.
+Typically, data in segments does not overlap. If the same data point is stored in multiple segments, it won't cause issues because the search process includes a deduplication mechanism.
 
-The configuration of the segments in the collection can be different and independent of one another, but at least one `appendable' segment must be present in a collection.
+Segments are made up of:
+- Vector and Payload Storage
+- Vector and Payload [Indexes](/documentation/concepts/indexing/)
+- An ID mapper that links internal and external IDs
 
-## Vector storage
+Segments can be either `appendable` or `non-appendable`:
+- **Appendable segments**: You can add, delete, and query data freely.
+- **Non-appendable segments**: You can only read and delete data.
```

Comment:
Payload data can also be updated in non-appendable segment. But maybe it's better to not mention that here.